### ISSUE TITLE
Include Events with Orientation Date Event Category in the Portal-Upcoming Events

### ISSUE THREAD

**Issue by dreazee** [SDRefugeeTutoring/sdrt-custom-functions#110]
So @JasonTheAdams as we embark on a new trimester, I'm just over here thinking about all the stuff to make our Portal better functioning! Thinking more how this Portal should really be the main source of information as far as events. Originally, it was chosen to not add Orientation Dates to the Portal but they really should be included - especially now that we have the fancy Event Types in there! 🥇 
Also, is it possible to add events with the Orientation Date Event Category to the Portal but NOT include a box above the display to allow users to filter for that type of event? I'm just thinking of a way to include it but not have it look too busy or crowded above the event display box. Do you think this would be a bad idea/design? I will defer to you.

**Comment by JasonTheAdams** [comment]
Hi @dreazee!

Thanks for sharing your idea! I've thought through it a bit and have a few comments.

My understanding of "Upcoming Events" is that the goal is to give volunteers a way of seeing and quickly RSVPing to events, especially sessions. Orientations are a bit different since we only want volunteers to RSVP "yes" to a single orientation a year; once they've attended an orientation they don't need to worry about it for another year. As such, it's not really valuable for volunteers to see orientations once they've attended one.

For this reason, the orientations were broken out into the Requirements page — where they can see the list of orientations and RSVP to the one they plan on attending. Once they've completed this requirement the orientations are hidden until the next year.

In short, it seems like it would be adding additional data to the volunteer when in reality its only meaningful to them once per year. Seems like it's worth keeping that separate to me.

Happy to hear your thoughts and continue the dialogue! 😄

**Comment by dreazee** [comment]
Hi @JasonTheAdams  I thought I responded to this but didn't! Yes this makes sense. HOWEVER.... I have another similar issue.

We are now going to have enrichment sessions for the tutors during the year and I want these to show up in the "Upcoming Events" so tutors see them when they RSVP for sessions. Definitely don't want them to miss them. 
So I created a new Event Category under the Parent Category of "Tutoring Dates" called "Tutor Enrichment", with the thought that they would show up in "Upcoming Events" since it is listed under Tutoring Dates (and not Orientation dates). However, the "Upcoming Events" widget (is it a widget?) is not coded to pull in this new sub-category. Does this make sense to you to do it like this? For now the [Jan 4 Tutor Enrichment session](https://sdrefugeetutoring.com/event/2024-tips-jan4/) is listed under the Event Category of "Non-Tutoring Event" so that it shows up in "Upcoming Events". But it's not a Non-Tutoring Event since it absolutely relates to tutoring since the tutors are being tutored by us!
